packages/flutter/test/widgets/slivers_padding_test.dart

```diff
@@ -618,7 +618,15 @@ void main() {
   });
 }
 
-void expectIsTextSpan(Object target, String text) {
-  expect(target, isA<TextSpan>());
-  expect((target as TextSpan).text, text);
+void hitsText(HitTestResult hitTestResult, String text) {
+  switch (hitTestResult.path.first.target) {
+    case final TextSpan span:
+      expect(span.text, text);
+    case final RenderParagraph paragraph:
+      final InlineSpan span = paragraph.text;
+      expect(span, isA<TextSpan>());
+      expect((span as TextSpan).text, text);
+    case final HitTestTarget target:
+      expect(target, anyOf(isA<TextSpan>(), isA<RenderParagraph>()));
```

Comment:
Shouldn't this case somehow use the `text` parameter? Otherwise, the method is kinda misleading if things can pass without verifying the `text` was hit.

Reply:
It's bound to fail once it reaches this case. Added a code comment.